src/ow1/role_limit.opy

```diff
@@ -0,0 +1,69 @@
+#!mainFile "../main.opy"
+
+#Global variables
```

Comment:
If you sourced this code from someone else, make sure you credit them with comments starting at line 3 (below the mainfile declarative)